docker/cluster/docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       amaru:
         condition: service_started
     environment:
-      - NETWORK
+      - AMARU_NETWORK
       - LEDGER_DIR
       - CONFIG_FOLDER
       - CHAIN_DIR
```

Comment:
@pgrange looks like those should reflect new env names? e.g. `PEER_ADDRESS` => `AMARU_PEER_ADDRESS` ?